app.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const app = express();
+const getTopics = require('./controller/controller')
+
+
+
+
+
+// GET /api/topics
+app.get('/api/topics',getTopics);
+
+
+
+app.use((err,req,res,next) => {
+
+    console.log(err)
+
+})
```

Comment:
This error handler currently logs the error but does not return anything. Your initial error handler at this point should return a 500 status for internal server error. A 404 error can also handle anything on the api endpoint that does not exist.